mutis/signal.py

```diff
@@ -184,12 +202,14 @@ def PSD_check_gen(self, fgen=None, ax=None):
         if ax is None:
             ax = plt.gca()
 
-        if fgen == "lc_gen_psd_nft":
+        if fgen == "lc_gen_psd_fft":
+            s2 = lc_gen_psd_fft(self.values)
+        elif fgen == "lc_gen_psd_nft":
             s2 = lc_gen_psd_nft(self.times, self.values)
         elif fgen == "lc_gen_psd_lombscargle":
             s2 = lc_gen_psd_lombscargle(self.times, self.values)
-        elif fgen == "lc_gen_psd_fft":
-            s2 = lc_gen_psd_fft(self.values)
+        elif fgen == "lc_gen_psd_c":
+            s2 = lc_gen_psd_c(self.times, self.values, self.times)
         else:
             raise Exception("No valid fgen specified")
```

Comment:
Why removing `lc_gen_psd_fft` ?

Reply:
`lc_gen_psd_fft` is not removed, only changed its place on the if-clause, so they are ordered?

Reply:
ok , understood 👍🏻